runtime-config.md

```diff
@@ -1,3 +1,8 @@
+# Host-specific container configuration
+
+The bundle's top-level directory MUST contain a configuration file called `runtime.json` with [UTF-8][] [JSON][].
+For now the canonical schema is defined in [`runtime_config.go`](runtime_config.go) and [runtime_config_linux.go](runtime_config.go), but this will be moved to a formal JSON schema over time.
```

Comment:
On Tue, Sep 08, 2015 at 03:59:13PM -0700, Vincent Batts wrote:
> This feels like something to have roadmapped and discussed rather
> than alluding to.

Sounds reasonable to me.  Shall I open the mailing-list thread, or
would you like to?

And is there an alternative phrasing you'd like to see here in the
meantime, so we can land the UTF-8 notes in this PR without waiting on
the desired-schema-language discussion?  Copying the existing phrasing
from config.md seemed like the least opinionated approach, but maybe
just dropping the “For now the canonical schema…” line with it's
Go-file links would be better?